src/store/getters.ts

```diff
@@ -107,11 +146,85 @@ export const getHasNoCorpNum = (state: StateIF): boolean => {
   return state.stateModel.newRequestModel.noCorpNum
 }
 
+export const getCorpNum = (state: StateIF): string => {
+  return state.stateModel.newRequestModel.corpNum
```

Comment:
Does `newRequestModel` make sense to you?
It should almost be called `everythingModel`... I'm kinda wondering if it's helpful to have this object or not. Thoughts?